gravitee-apim-rest-api/gravitee-apim-rest-api-portal/gravitee-apim-rest-api-portal-rest/src/main/resources/openapi.yaml

```diff
@@ -1580,6 +1580,42 @@ paths:
                 $ref: '#/components/schemas/ErrorResponse'
         500:
           $ref: '#/components/responses/InternalServerError'
+  /applications/{applicationId}/keys/{apiKey}/_revoke:
+    parameters:
+      -   $ref: '#/components/parameters/applicationIdParam'
+      -   name: apiKey
+          in: path
+          required: true
+          schema:
+            type: string
+    post:
+      tags:
+        - Application
+      summary: Revoke the shared api key of on application.
+      description: |
+        Revoke the shared api key of on application.
+        The application must have the ApiKeyMode set to SHARED
+        User must have APPLICATION_SUBSCRIPTION[UPDATE] permission.
+      operationId: revokeSharedKey
+      responses:
+        204:
+          description: No-Content
+        400:
+          description: Application does not use shared API key mode
```

Comment:
Not sure this error really happens currently, cause the same service is called for revoking an API key (shared, or non-shared)